docker-action.yml.mustache

```diff
@@ -24,6 +24,9 @@ jobs:
 {{# tested_coq_opam_versions }}
           - '{{ repo }}{{^ repo }}coqorg/coq{{/ repo }}:{{ version }}'
 {{/ tested_coq_opam_versions }}
+{{# tested_rocq_opam_versions }}
+          - '{{ repo }}{{^ repo }}rocq/rocq-prover{{/ repo }}:{{ version }}'
```

Comment:
The `repo` field is used for specifying both Coq and Rocq repo's. Not sure if it is desired.